common/xrdp_client_info.h

```diff
@@ -128,6 +128,9 @@ struct xrdp_client_info
 
   char certificate[1024];
   char key_file[1024];
+  char rsakeys_ini_file[1024];
+  char xrdp_keyboard_ini_file[1024];
+  char keymaps_path[1024];
```

Comment:
is it a problem? touching any file in common/ resulted in xordxrdp recompilation.